README.md

```diff
@@ -1 +1,55 @@
-# filestorage
\ No newline at end of file
+### File storage
+`new FileStorage(dir)`
+  - `dir` - string, data storage path
+
+Example:
+```js
+const { FileStorage } = require('filestorage');
+const storage = new FileStorage('./root-directory');
```

Comment:
Could you also add an example with `create`
```js
create('./root-directory', (err, storage) => {
  ...
});
```
And a note that `create` will create directories if needed but clean `FileStorage` expects them to exist.